auth/src/main/kotlin/com/hedvig/android/auth/NetworkAuthRepository.kt

```diff
@@ -26,6 +27,7 @@ class NetworkAuthRepository(
 
   private val jsonBuilder = Json {
     ignoreUnknownKeys = true
+    explicitNulls = false
```

Comment:
Which part of the API responds without a specific key at all and we need to infer it as null? Was it the otp stuff? Is that potentially an error on the backend side or is it expected that it doesn't send that thing at all instead of sending it as null?

Reply:
This is because the response can omit some fields when we get error responses.